server/db/index.js

```diff
@@ -7,19 +7,32 @@ const client = new Client();
 
 
 client.connect();
-client.end();
-//   .then(() => console.log('connected'))
-//   .catch(e => console.error('connection error', err.stack));
+
+client.on('end', () => {
+  console.log('pg client end');
+});
+
+client.on('error', (error) => {
+  console.error('pg client error', error);
+});
 
 const getOpenSeats = (restaurantId, date) => {
```

Comment:
Not quite sure what this function is doing, maybe i'm missing something but it is just a console log and returning 0, and the test is just to see if it is a function. Will this be an actual function to get open seats later?

Reply:
Same query here as Namson - assume that this function is more or less a place holder at this time?

Reply:
Yeah, getOpenSeats() is still just a placeholder function for now.  The console.log() is just so my linter doesn't error because of unused parameters.